be/src/olap/olap_header_manager.cpp

```diff
@@ -126,19 +126,7 @@ OLAPStatus OlapHeaderManager::set_converted_flag(OlapStore* store) {
 }
 
 OLAPStatus OlapHeaderManager::traverse_headers(OlapMeta* meta,
-        std::function<bool(long, long, const std::string&)> const& func) {
-    auto traverse_header_func = [&func](const std::string& key, const std::string& value) -> bool {
-        std::vector<std::string> parts;
-        // key format: "hdr_" + tablet_id + "_" + schema_hash
-        split_string<char>(key, '_', &parts);
-        if (parts.size() != 3) {
-            LOG(WARNING) << "invalid header key:" << key << ", splitted size:" << parts.size();
-            return true;
-        }
-        TTabletId tablet_id = std::stol(parts[1].c_str(), NULL, 10);
-        TSchemaHash schema_hash = std::stol(parts[2].c_str(), NULL, 10);
-        return func(tablet_id, schema_hash, value);
-    };
```

Comment:
You can't change it, this encapsulate `tablet_id` and `schema_hash` encoding logic. If you put it logic outside, every one need to implement the same logic

Reply:
OK, I will reset it to last version.